src/e2e-test/java/io/cdap/cdap/ui/utils/Commands.java

```diff
@@ -67,6 +67,23 @@ public static void connectTwoNodes(NodeInfo source, NodeInfo target) {
       getNode(target));
   }
 
+  public static void openNodeProperty(NodeInfo node) {
+    String elementId = "plugin-node-" + node.getNodeName() + "-"
+            + node.getNodeType() + "-" + node.getNodeId();
+    WebElement nodeElement = Helper.locateElementByTestId(elementId);
+    WebElement versionElement = nodeElement.findElement
+            (By.cssSelector("div[class*='node'] > div[class*='node-metadata'] > div[class*='node-version']"));
+
+    JavascriptExecutor jsExecutor = (JavascriptExecutor) SeleniumDriver.getDriver();
+    jsExecutor.executeScript("arguments[0].style.visibility='hidden'", versionElement);
```

Comment:
what are these two scripts doing?

Reply:
You can use `CdfStudioActions.navigateToPluginPropertiesPage(node.getNodeName());` to open a node property page by its name;

Reply:
Those two scripts are not needed anymore as method is replaced with `CdfStudioActions.navigateToPluginPropertiesPage`.